src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
+const spanList = document.getElementsByClassName('population');
+let summ = 0;
+let avg = 0;
+
+if (spanList.length > 0) {
+  summ = [...spanList].reduce(
+    (accumulator, currentValue) =>
+      accumulator + parseInt(currentValue.textContent.replaceAll(',', '')),
+    0
+  );
+
+  avg = summ / spanList.length;
+}
+
+document
+  .querySelector('.total-population')
```

Comment:
it's better to create own variables for `document.querySelector` it will make you code clearer

Reply:
matter of argument